modules/serverless-functions-func-yaml-environment-variables.adoc

```diff
@@ -0,0 +1,28 @@
+[id="serverless-functions-func-yaml-environment-variables_{context}"]
+= Referencing local environment variables from func.yaml fields
+
+In any of the fields in `func.yaml`, you can put a reference to an environment variable available in the local environment. This can be useful for avoiding storing sensitive information such as an API key in the function configuration.
```

Comment:
```suggestion
In the `envs` field of `func.yaml`, you can put a reference to an environment variable available in the local environment. This can be useful for avoiding storing sensitive information such as an API key in the function configuration.
```
Something like this works imho^, but feel free to polish it. This local env is only applicable to `envs` field.

Can we link this section in the item 2 in the `envs` section? Something like "for more details please refere to..." etc

Reply:
Thanks, used your wording and linked from item 2 (let's hope it passes the peer review).

Reply:
@rh-max I don't see the link anywhere in the PR?

Reply:
@zroubalik Gah, sorry, push failed due to some accepted suggestions. It's here now: https://github.com/openshift/openshift-docs/pull/34442/commits/0e1c7a2c3c513645971c5316950f520d7d2f37bb (No actual link is by design - remember that we can't link from anywhere except "Additional resources" :/ )